occipital.py

```
@@ -84,6 +101,9 @@ def _build_model(self):
         return model
 
     def save_model(self):
```

Ruff (0.12.2)

Missing return type annotation for public function `save_model`

Add return type annotation: `None`

(ANN201)